.eslintrc

```diff
@@ -1,9 +1,13 @@
 {
  "extends": "./node_modules/lab/lib/linters/eslint/.eslintrc",
- "parser": "babel-eslint",
+ "env": {
```

Comment:
The `es6` part is to get the ES6 features turned on. The `node` part is just for good measure.